README.md

```diff
@@ -70,7 +70,46 @@ The following camera and microphone permissions need to be granted to enable aud
 
 More details on how to request permissions can be found in [Android](https://developer.android.com/training/permissions/requesting) and [iOS](https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/requesting_authorization_for_media_capture_on_ios?language=objc) official documents.
 
-## 3. Connect React Native code to the Amazon Chime SDK
+## 3. Add libraries for SDK
+
+### Android
```

Comment:
Only android?

Reply:
iOS doesn't seem to be have extra dependencies